lib/features/support/maze_runner.rb

```diff
@@ -7,8 +7,8 @@
 class MazeRunner
   class << self
     attr_accessor :driver
-    def configuration
-      @configuration ||= Configuration.new
+    def config
```

Comment:
Not concerned at all, just wondering the reason behind this change?

Reply:
Just for brevity, that's all.  Some areas of the code are a bit noisy currently and it was easy to make this change now that there are no notifiers currently making use of it.